mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -135,7 +132,8 @@ impl CommandHandler {
 
     async fn handle_event(&mut self, event: Event) -> Result<(), HandleDisconnectError> {
         if let Event::Publication(publication) = event {
-            let client_id = parse_client_id(&publication.topic_name)?;
+            let client_id = str::from_utf8(&publication.payload)
+                .map_err(HandleDisconnectError::ParseClientId)?;
```

Comment:
EdgeHub will only disconnect one client at a time

Reply:
For the sake of minimizing communication, should we be ready to process a list of client ids?

Reply:
I was actually thinking of doing this but seemed like a premature optimization and also adds unnecessary parse logic (which is one of the things I didn't like about the regex). I thought it was a good idea to not do it now, but if you disagree I will add it.

Reply:
Wait, what parsing you are talking about? As far as I can see, it's not a JSON, but it should be